Pod/Classes/VOKCoreDataManager.m

```diff
@@ -112,13 +127,23 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator
     return _persistentStoreCoordinator;
 }
 
+- (NSBundle *)bundleForModel
+{
+    if (!_bundleForModel) {
+        //Default to using the bundle for this class.
+        _bundleForModel = [NSBundle bundleForClass:[self class]];
+    }
+    
+    return _bundleForModel;
```

Comment:
I'd suggest that you `return _bundleForModel ?: [NSBundle bundleForClass:[self class]];` instead, so that the property is null-resettable.

Also, is using the bundle for the pod a better default than using the main bundle?  (I really don't know.)